browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -1073,22 +1073,26 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			var container = L.DomUtil.create('div', 'ui-frame-container ' + builder.options.cssClass, parentContainer);
 			container.id = data.id;
 
-			var frame = L.DomUtil.create('div', 'ui-frame ' + builder.options.cssClass, container);
-			frame.id = data.id + '-frame';
-			var label = L.DomUtil.create('label', 'ui-frame-label ' + builder.options.cssClass, frame);
-			label.innerText = builder._cleanText(data.children[0].text);
-			label.id = data.children[0].id;
-			if (data.children[0].visible === false)
-				L.DomUtil.addClass(label, 'hidden');
-			builder.postProcess(frame, data.children[0]);
+			var isLabel = data.children[0].type === 'fixedtext';
+			if (isLabel) {
```

Comment:
we always should create the same label here, no matter if it is simple fixed text or test inside container.

Please prepare function for `extractLabelText(data.children[0])` which will return correct text, and use it for `label.innerText`